services/collaboration/pkg/connector/fileconnector.go

```diff
@@ -390,15 +428,15 @@ func (f *FileConnector) RefreshLock(ctx context.Context, lockID string) (*Connec
 				Str("StatusCode", resp.GetStatus().GetCode().String()).
 				Str("StatusMsg", resp.GetStatus().GetMessage()).
 				Msg("RefreshLock failed, no lock on file")
-			return NewResponseWithLock(409, ""), nil
+			return NewResponseConflictWithVersion(statResp.GetInfo().GetMtime(), "No lock on file"), nil
```

Comment:
We probably should send the lockID in case of conflict, even if it's an empty lockID

Reply:
correct